src/Microsoft.Health.Fhir.SqlServer/Features/Schema/SchemaVersionConstants.cs

```diff
@@ -7,9 +7,9 @@ namespace Microsoft.Health.Fhir.SqlServer.Features.Schema
 {
     public static class SchemaVersionConstants
     {
-        public const int Min = (int)SchemaVersion.V63;
-        public const int Max = (int)SchemaVersion.V63;
-        public const int MinForUpgrade = (int)SchemaVersion.V58; // this is used for upgrade tests only
+        public const int Min = (int)SchemaVersion.V64;
```

Comment:
Should be v60, or no greater than Max-1.

Reply:
I don't think I can make new code to work with old database schema.

Reply:
Addressed